internal/contour/cachehandler.go

```diff
@@ -19,19 +19,25 @@ package contour
 import (
 	"time"
 
+	xds "github.com/envoyproxy/go-control-plane/pkg/cache/types"
 	"github.com/projectcontour/contour/internal/dag"
 	"github.com/projectcontour/contour/internal/metrics"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/sirupsen/logrus"
 )
 
-// CacheHandler manages the state of xDS caches.
+// CacheHandler manages the state of xDS translators.
```

Comment:
I wonder if it would make sense to combine this struct with the SnapshotHandler. It's not doing a whole lot anymore, seems like it may be an unnecessary layer of indirection. The EventHandler could directly call into the SnapshotHandler if we got rid of this one.

Reply:
Yup that's a good point. I was taking it at baby steps, but like you said, it doesn't need to stay that way. Let me refactor a bit more.